pihole

```diff
@@ -300,6 +300,29 @@ Branches:
   checkout "$@"
 }
 
+tricorderFunc() {
+  if [ ! -p "/dev/stdin" ]; then
+    echo "Please do not call Tricorder directly."
+    exit 1
+  fi
+
+  if command -v openssl &> /dev/null; then
```

Comment:
Oh, in the event that a local firewall is blocking the port? That's a good thought.